packages/expo/src/generators/library/schema.json

```diff
@@ -88,6 +88,11 @@
       "type": "boolean",
       "description": "Whether or not to configure the ESLint \"parserOptions.project\" option. We do not do this by default for lint performance reasons.",
       "default": false
+    },
+    "skipPackageJson": {
+      "type": "boolean",
+      "description": "Do not add packages to `package.json` (e.g., `--skipPackageJson`).",
```

Comment:
also, you could remove the ' (e.g., `--skipPackageJson`)' part from description?

Reply:
Gladly! 

I had just grabbed an example from somewhere else in the app -- but yeah it does seem a little redundant 
![image](https://user-images.githubusercontent.com/2213636/215253179-4bc099e6-91c8-48e7-9fe9-d9686c52176a.png)